asm/non_matchings/audio/external/func_800C2274.s

```diff
@@ -21,8 +21,8 @@ glabel func_800C2274
 /* 0C2EC0 800C22C0 AFA40048 */  sw    $a0, 0x48($sp)
 /* 0C2EC4 800C22C4 13200062 */  beqz  $t9, .L800C2450
 /* 0C2EC8 800C22C8 2406000F */   li    $a2, 15
-/* 0C2ECC 800C22CC 3C02800E */  lui   $v0, %hi(D_800DC530) # $v0, 0x800e
-/* 0C2ED0 800C22D0 8C42C530 */  lw    $v0, %lo(D_800DC530)($v0)
+/* 0C2ECC 800C22CC 3C02800E */  lui   $v0, %hi(gScreenModeSelection) # $v0, 0x800e
```

Comment:
What does screenMode mean?

Reply:
It controls how the screen is displayed during gameplay and the demos. So whether the screen is split for 2 players, 3+ players, or just 1 player.... also where the pause menu text appears.